AcademyResidentInformationApi.Tests/V1/E2ETests/GetClaimantInformationById.cs

```diff
@@ -27,7 +27,7 @@ public async Task GetClaimantInformationByIdReturnsTheCorrectInformation()
             var academyId = $"{claimId}-{personRef}";
             var expectedResponse = E2ETestHelpers.AddPersonWithRelatesEntitiesToDb(AcademyContext, academyId);
```

Comment:
You could probably replace academy Id and pass in `claimId` and `personRef` as arguments to this method.
I don't think we need to keep Academy Id as a variable in the tests.

Reply:
This would be part of the new ticket I think.  This works around the current implementation of the API where the response returns an academyId value in the response

Reply:
@LBHBomotoso and I have decided to have these changes that deal with the academy ID in a separate PR.